# Request fields
int8   interaction_id
bool   return_full_trajectory
uint32 num_samples
---
# Response fields
Trajectory generated_trajectory
